src/components/link/index.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+
+const COMPONENT_CLASS_NAME = 'c-link';
+
+const ExternalLink = ({ children, href }) => (
+	<a
+		className={COMPONENT_CLASS_NAME}
+		href={href}
+		rel="noreferrer"
+		target="_blank"
+	>
+		<span className="visually-hidden">Opens in new window</span>
```

Comment:
Couple of points

* The visually hidden text should come after the link contents
* The text should be passed in as a prop to account for translation support - we can fall back to English here to ensure it's output as needed